### DISCUSSION TITLE
能不能加个安静模式啊?

### DISCUSSION THREAD

**Discussion by Milanda007** [nilaoda/N_m3u8DL-CLI#582] [General]
能不能加个安静模式啊?

**Comment by nilaoda** [comment]
Playlist is encrypted.

Relative code here:

![image](https://user-images.githubusercontent.com/20772925/147735688-62288dd7-6526-4894-9a54-3db2f6a4e672.png)

Before you watch this video, add break point here(`Vis2.js line 4658`), then click play button, you can get the clean playlist:

![image](https://user-images.githubusercontent.com/20772925/147735994-d2f5769a-f269-42c9-af5c-ce52501ffcd6.png)

and excute follow code:
```js
var blob = new Blob([n.replaceAll(/(\s)\/(.*)/g,"$1https://stream.uploadvi.ir/$2")], {
	type: "text/plain"
});
var url = URL.createObjectURL(blob);
var title = "v.m3u8";
var aLink = document.createElement("a");
aLink.href = url;
aLink.download = title;
aLink.style.display = "none";
var event;
if (window.MouseEvent) {
	event = new MouseEvent("click")
} else {
	event = document.createEvent("MouseEvents");
	event.initMouseEvent("click", true, false, window, 0, 0, 0, 0, 0, false, false, false, false, 0, null)
}
aLink.dispatchEvent(event);
```

browser will download a file name `v.m3u8`

then add header `referer: https://stream.uploadvi.ir`

command like this
```
N_m3u8DL-CLI.exe "C:\Downloads\v.m3u8" --saveName "v" --headers "referer: https://stream.uploadvi.ir" 
```

![image](https://user-images.githubusercontent.com/20772925/147739875-d433499d-1292-4867-beac-1521831abc91.png)
